airqo-platform/AirQo-frontend
# Mob 103 development project

#### Summary of Changes (What does this PR do?)
- Creating a staging environment for the mobile app for Android.
- Flavors were used, i.e a production flavor and a dev flavor.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Get respective googleservices.json file from the firebase console for both airqo and airqodev projects
* For airqodev, place the googleservices.json file in (`app/src/airqodev`). For airqo, place the googleservices.json file in (`app/src/airqo`)
* Run the app locally for both flavors and for both environments, using the following commands, to check if all functionality is working well.
  *  Staging in debug mode.
 `flutter run --flavor airqodev -t lib/main_dev.dart`
  *  Staging in release mode.
 `flutter run --flavor airqodev -t lib/main_dev.dart` --release
  * Production in debug mode.
 `flutter run --flavor airqo -t lib/main.dart`
  * Production in release mode.
`flutter run --flavor airqo -t lib/main.dart` --release
* Inspect the code to suggest any improvements.
#### What are the relevant tickets?
  - [MOB-103](https://airqoteam.atlassian.net/browse/MOB-103)

#### Screenshots (optional)
